script.js

```diff
@@ -96,6 +107,67 @@ $(function(){
   })
 })
 
+$(function(){
+  $("#newuser_form").submit(function(e){
+    e.preventDefault()
+    inputone = $("#input1").val()
+    inputtwo = $("#input2").val()
+    $.ajax({
+      type:"POST",
+      url:"04_AddUser.php",
+      data:{input1:inputone, input2:inputtwo},
+      success: function(data){
+        if (data.trim() != ""){
+          document.getElementById("newuser_box_message").innerHTML = data
+        }
+        else{
+          document.getElementById("newuser_box_message").innerHTML = data
+          window.location.reload()
+        }
+      }
+    })
+  })
+})
```

Comment:
Allows an error message for forms to display on the form itself and prevents the page from refreshing.